var/spack/repos/builtin/packages/sollve/package.py

```diff
@@ -0,0 +1,730 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sollve(CMakePackage):
+    """The SOLLVE Project is a collection of modular and reusable compiler and
+       toolchain technologies. Despite its name, SOLLVE has little to do
+       with traditional virtual machines, though it does provide helpful
+       libraries that can be used to build them. The name "LLVM" itself
+       is not an acronym; it is the full name of the project.
+    """
+
+    homepage = 'https://github.com/SOLLVE/' 
+    git = 'https://github.com/SOLLVE/sollve.git'
+    list_url = homepage
```

Comment:
You can remove this, Spack already knows where to find the `list_url` on GitHub.

Reply:
Thanks for your suggestion. We removed `list_url`.